stix_shifter_modules/msatp/stix_translation/json/from_stix_map.json

```diff
@@ -3,58 +3,61 @@
     "fields": {
```

Comment:
all these changes need to go in stix2.1 mappings as well- `/msatp/stix_translation/json/stix_2_1/from_stix_map.json`

Reply:
done